Core/src/org/sleuthkit/autopsy/casemodule/services/Services.java

```diff
@@ -93,12 +94,25 @@ public KeywordSearchService getKeywordSearchService() {
 
     /**
      * Gets the blackboard service for the current case.
+     * 
+     * @deprecated use org.sleuthkit.autopsy.casemodule.services.getCaseBlackboard()
+     * instead
      *
      * @return The blackboard service for the current case.
      */
+    @Deprecated
     public Blackboard getBlackboard() {
         return blackboard;
     }
+    
+     /**
+     * Gets the case blackboard service for the current case.
+     *
+     * @return The blackboard service for the current case.
+     */
+    public Blackboard getCaseBlackboard() {
+        return blackboard;
+    }
```

Comment:
This method is returning org.sleuthkit.autopsy.casemodule.services.Blackboard. It should return the TSK Blackboard.